lib/Fhp/Action/SendSEPATransfer.php

```diff
@@ -49,7 +49,7 @@ protected function createRequest(BPD $bpd, ?UPD $upd)
     {
         //ANALYSE XML FOR RECEIPTS AND PAYMENT DATE
         $xmlAsObject = simplexml_load_string($this->painMessage, "SimpleXMLElement", LIBXML_NOCDATA);
-        $payments = count($xmlAsObject->CstmrCdtTrfInitn?->PmtInf);
+        $numberOfTransactions = $xmlAsObject->GrpHdr->NbOfTxs;
```

Comment:
$numberOfTransactions = $xmlAsObject->**CstmrCdtTrfInitn**->GrpHdr->NbOfTxs;

Könntest du die Änderungen noch mal mit Überweisungen testen?

Reply:
Ist gerade schwierig... Ich habe die Änderung nur in meinem GitHub gemacht. Ich bin die nächsten Wochen unterwegs und habe eig. nur unsere ProduktivUmgebung - also kein Test möglich

Reply:
Ist nun implementiert und getestet. Alle 4 Arten kommen bei der Bank an.

Reply:
Danke!